### DISCUSSION TITLE
Why not add `nvm` if there is already `nvm-windows`?

### DISCUSSION THREAD

**Discussion by noftaly** [sindresorhus/awesome-nodejs#1096] [Ideas]
Hey!

I was wondering, why is only `nvm-windows` being listed, but not the original [`nvm` (for Unix)](https://github.com/nvm-sh/nvm)?
Is it just because you didn't know about it or should I make a PR?

**Comment by sindresorhus** [comment]
There are better version managers available for macOS and Linux. For Windows, there are (or were) not that many options, hence `nvm-windows` was allowed (which is a completely different project from `nvm`).

**Comment by noftaly** [comment]
Ok, thanks for the fast response 🙂

**Comment by lifeart** [comment]
should both be replaced with https://volta.sh/ :)
